tracer/src/Datadog.Trace/Iast/SourceType.cs

```diff
@@ -50,8 +51,9 @@ internal static string GetString(SourceTypeName value)
             SourceTypeName.RequestQuery => "http.request.query",
             SourceTypeName.RoutedParameterValue => "http.request.path.parameter",
             SourceTypeName.MatrixParameter => "http.request.matrix.parameter",
-            SourceTypeName.CookieName => "http.cookie.name",
-            SourceTypeName.CookieValue => "http.cookie.value",
+            SourceTypeName.CookieName => "http.request.cookie.name",
+            SourceTypeName.CookieValue => "http.request.cookie.value",
+            SourceTypeName.RequestUri => "http.request.uri",
```

Comment:
Is this `SourceTypeName.RequestUri` actually used anywhere 🤔 Should it be in a snapshot test somewhere?

Reply:
There is a task for it and will used shortly in the next PR. I just wanted in this one to stay syncronized with the other teams json.